src/components/erc20/erc20_app.tsx

```diff
@@ -19,28 +21,46 @@ const AccountTradingsPage = lazy(() => import('./pages/account_trading'));
 const Marketplace = lazy(() => import('./pages/marketplace'));
 const WizardPage = lazy(() => import('./pages/wizard'));
 
-const Erc20App = () => {
+const messages: any = {
```

Comment:
Add correct type here instead of using any. create an interface of Languages like interface Message { [key: string]: string }